docs/parts/wokwi-hx711.md

```diff
@@ -0,0 +1,97 @@
+---
+title: wokwi-hx711 Reference
+sidebar_label: wokwi-hx711
+---
+
+HX711 Load Cell Amplifier
+
+<wokwi-hx711 type="50kg" />
+
+## Pin names
+
+| Name | Description                                       |
+| ---- | ------------------------------------------------- |
+| VCC  | Voltage supply (5V)                               |
+| DT   | Serial data                                       |
+| SCK  | Serial clock                                      |
+| GND  | Ground                                            |
+
+*note that `E+`/`E-`/`A+`/`A-`/`B+`/`B-` pins are non-interactive and rendered based on attributes*  
+
+<wokwi-hx711 type="5kg" />
+
+## Attributes
+
+| Name     | Description                                | Default value |
+| -------- | ------------------------------------------ | ------------- |
+| type     | Either "50kg" (default), "5kg", or "guage" | "50kg"        |
+
+## Operation
+
+The HX711 amplifier allows you to easily read load cells and evaluate changes in resistance. A Wheatstone bridge is used to connect load cells to the IC, which is in turn connected to the microcontroller via `VCC`, `DT`, `SCK`, and `GND`. Use `begin()` to initialize the scale and `set_scale()` and `tare()` to calibrate it. `power_down()` and `power_up()` can be used to bring the ADC into and out of low power mode. `get_value()` and `get_units()` are used to read the ADC minus tare and divided, passing an optional integer value to obtain that number of values, averaged.
```

Comment:
It's worth mentioning that all these methods are part of the `HX711` library for Arduino (I can see there are several Arduino libraries for interfacing with the HX711).